src/timeline/TimelineModel.cpp

```diff
@@ -1934,11 +1934,279 @@ TimelineModel::formatPowerLevelEvent(const QString &id)
     if (!event)
         return QString();
 
-    QString user = QString::fromStdString(event->sender);
-    QString name = utils::replaceEmoji(displayName(user));
+    mtx::events::StateEvent<mtx::events::state::PowerLevels> *prevEvent = nullptr;
+    if (!event->unsigned_data.replaces_state.empty()) {
+        auto tempPrevEvent = events.get(event->unsigned_data.replaces_state, event->event_id);
```

Comment:
This needs to be scoped outside of the if, otherwise prevEvent will be dangling.

Reply:
Ah, I guess it is fine, because this is a global pointer stored in the cache... Hm

Reply:
Its also copied from one of the other functions in that file :P